lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1600,7 +1599,7 @@ def test_course_image_for_split_course(self, store):
         self.course.course_image = ''
 
         url = course_image_url(self.course)
-        assert 'static/test.png' == url
+        assert '/static/test.png' == url
```

Comment:
In `Django 3.1+`, `STATIC_URL` is automatically prepended with `/` if it is not present at the start of the relative path. See details: https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-STATIC_URL.